<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="2" hostname="DELL-PC" name="com.bankguru.user.Level_17_Custom_Close_Browser_Driver" tests="3" failures="1" timestamp="21 Nov 2020 08:54:40 GMT" time="13.726" errors="0">
  <testcase name="beforeClass" time="13.726" classname="com.bankguru.user.Level_17_Custom_Close_Browser_Driver">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.bankguru.user.Level_17_Custom_Close_Browser_Driver.beforeClass(Level_17_Custom_Close_Browser_Driver.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:40)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:489)
at org.testng.internal.thread.ThreadUtil$1.call(ThreadUtil.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- beforeClass -->
  <testcase name="TC_01_Register_To_System" time="0.000" classname="com.bankguru.user.Level_17_Custom_Close_Browser_Driver">
    <skipped/>
  </testcase> <!-- TC_01_Register_To_System -->
  <testcase name="TC_02_Login_To_System" time="0.000" classname="com.bankguru.user.Level_17_Custom_Close_Browser_Driver">
    <skipped/>
  </testcase> <!-- TC_02_Login_To_System -->
</testsuite> <!-- com.bankguru.user.Level_17_Custom_Close_Browser_Driver -->
